examples/TwoHundredFields.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Formik, FastField, Form } from 'formik';
+import { unstable_trace as trace } from 'scheduler/tracing';
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+const onSubmit = async values => {
+  await sleep(300);
+  window.alert(JSON.stringify(values, 0, 2));
+};
+
+const Basic = ({ numFields = 200 }) => {
+  const keys = [...Array(numFields).keys()];
+  const initialValues = keys.reduce((prev, curr) => {
+    prev['field' + curr] = '';
+    return prev;
+  }, {});
+  return (
+    <Formik onSubmit={onSubmit} initialValues={initialValues}>
+      <Form>
+        <h1>Formik v2.x - {numFields} controlled fields</h1>
+        {keys.map(key => (
+          <FastField name={`field${key}`} key={key}>
```

Comment:
Um, what's the point of using FastField? It's just renamed export of Field, right? :) It's probably misleading that regular Field would be bad for this use case.

Reply:
It's coming back. I deleted it but just didn't change the example